e2e/src/usecases/DappListDisplay.js

```diff
@@ -32,19 +40,19 @@ export default DappListDisplay = () => {
   })
 
   it('should show dapp bottom sheet when dapp is selected', async () => {
-    await element(by.id('DappCard')).atIndex(0).tap()
+    await element(by.id('DappCard')).atIndex(dappToTest.index).tap()
```

Comment:
Why not finding the index from `impactMarket` here?
I'm slightly concerned the `fetchDappList` order could be different since it doesn't pass the wallet address. And we've done experiments in the past changing the order based on it.

Reply:
Or why not incrementing the index until we get something that's not Bidali?
This way we don't have to rely too much on the fetchDappList data.